src/common/utils.js

```diff
@@ -214,3 +220,23 @@ export const readMnemonicFromFile = async () => {
 };
 
 export const calculateTimeout = (executionTime) => (executionTime - moment().valueOf() / 1000 + LISTEN_EVENT_DELAY) * 1000;
+
+export const askScheduleAction = async () => {
+    const actions = [
+        {
+            name: 'Execute immediately',
+            value: ScheduleActionType.executeImmediately,
+            description: 'Execute task immediately.',
+        },
+        {
+            name: 'Execute on the hour',
+            value: ScheduleActionType.executeOnTheHour,
+            description: 'Execute task on the hour.',
+        },
+    ];
+    const actionSelected = await select({
+        message: 'Select an action to perform',
+        choices: actions,
+    });
+    return actionSelected;
+};
```

Comment:
This is only used for Mangata Dev. Why do you extract into utils? Put it back into src/mangata.js